ci/run_cuml_accel_integration_pytests.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# Copyright (c) 2025, NVIDIA CORPORATION.
+
+# Support invoking run_cuml_singlegpu_pytests.sh outside the script directory
+cd "$(dirname "$(realpath "${BASH_SOURCE[0]}")")"/../python/cuml/cuml/tests
+
+python -m pytest -p cudf.pandas -p cuml.experimental.accel --cache-clear "$@"  experimental/accel/
```

Comment:
For the `cuml.accel` tests in `run_cuml_singlegpu_accel_pytests.sh` we `cd` into the `experimental/accel` directory. Can we keep it consistent? Not sure but maybe it effects the paths we see in the logs?